README.md

```diff
@@ -1 +1,30 @@
-Some sameple file
+# Celebro
+
+Celebro is a Cloud Based Event Management Application
+
+# Features
```

Comment:
Can we add instructions on setting up project locally?

Reply:
Will do.

Reply:
It is a Base Setup. Will update README in another Branch